std/concurrency.d

```diff
@@ -963,6 +963,50 @@ bool register(string name, Tid tid)
 }
 
 /**
+Retrieves an array containing all names associated with $(LREF Tid).
+These names have all been registered with the $(LREF register) function.
+
+Params:
+    tid = The $(LREF Tid) to locate within the registry
+
+Returns:
+    An array of associated names, or `null` if `tid` is not registered.
+
+See_Also:
+    $(LREF register)
+*/
+string[] registeredNames(Tid tid)
+{
+    synchronized (registryLock)
+    {
+        return namesByTid[tid];
+    }
+}
+
+///
+@system unittest
+{
+    static void spawnedFunc(Tid ownerTid)
```

Comment:
FWIW what I meant last time is that `ownerTid` is a [global function](https://dlang.org/phobos/std_concurrency.html#.ownerTid) - no need to pass it explicitly ;-)